### PR TITLE
update php dependencies for php 8

### PR DESCRIPTION
**Pull request by ratan99**

### CODE REVIEW

**Comment by jordan-globalexam** [comment]
Hello,
I need this update too.
Thanks a lot.

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "adaojunior/passport-social-grant",
+    "name": "atari/passport-social-grant",
```
Comment (kslimani) [inline composer.json:2]: Why this change ?

Comment (maximepvrt) [reply composer.json:2]: same question
